tests/src/rules/order.js

```diff
@@ -335,10 +335,10 @@ ruleTester.run('order', rule, {
     // Option newlines-between: 'always' with multiline imports #3
     test({
       code: `
-        import foo
+        import a
```

Comment:
why does this need to be changed? can this test be reverted?

Reply:
I think the intention of this commit (which is not my own) was to show how the alphabetical sorting in a clearer way, with minimal examples. I would not revert that.

Reply:
If your comment is concerned about only this particular test case (which tests multiline, not ordering), I will revert that part to its original state.

Reply:
Yes, my comment is because for it not to be a breaking change, no existing test cases should be altered.